### ISSUE TITLE
Google Maps API pricing

### ISSUE THREAD

**Issue by nicholaspad** [PrincetonUSG/food4u#22]
Hi team -
Due to USG's new budget policies, we'll have to cap food4u's Google Maps API usage so that the credit card on file never gets billed. I see on https://mapsplatform.google.com/pricing/ that you get $200 per month of free requests, which is 100k requests in total. Is this correct? Are you using the Static Maps API?

**Comment by benchanjamin** [comment]
@nicholaspad We're using the Maps JavaScript API that caps at around 28K-29K total requests for $200.
<img width="1021" alt="image" src="https://user-images.githubusercontent.com/47726262/153516228-f4ec8319-a930-4dbb-a34f-222c30e543cb.png">

**Comment by benchanjamin** [comment]
Yes, I am able to put a hard cap! Also, the map doesn't show up if I hit the cap.

![4205D08C-D65F-4A4E-8F87-A925A5B1F221](https://user-images.githubusercontent.com/47726262/153777301-f8c653d1-6391-445e-91a6-deb8d8cafde8.jpeg)

**Comment by nicholaspad** [comment]
Got it. Could you please put the hard cap if you haven't already?

**Comment by benchanjamin** [comment]
Capped it!
